tests/ui/rfcs/rfc-2632-const-trait-impl/trait-where-clause.rs

```diff
@@ -5,8 +5,12 @@ trait Bar {}
 
 trait Foo {
     fn a();
-    fn b() where Self: ~const Bar;
+    fn b()
+    where
+        Self: ~const Bar;
```

Comment:
Please change the formatting back to what it was (autoformatting should be disabled for ui tests)